components/blitz/src/ome/services/blitz/impl/OmeroMetadata.java

```diff
@@ -197,6 +197,8 @@ public String handleLsid(IObject obj) {
             Class<? extends IObject> k = obj.getClass();
             if (Annotation.class.isAssignableFrom(k)) {
                 k = Annotation.class;
+            } else if (Shape.class.isAssignableFrom(k)) {
```

Comment:
That'd be my guess; at least, something having the same effect. I went with this version because somebody already figured it for annotation and it seems to work. In the longer term I do have hope that `OmeroMetadata` could be generated automatically but that's the kind of project that could well take a month instead of a week.